### DISCUSSION TITLE
NotImplementedError: Don't know how to literal-quote value datetime

### DISCUSSION THREAD

**Discussion by NikitaAleshnikov** [collerek/ormar#209] [Q&A] [answered]
Please help with error(

Model:
class Contest(ormar.Model):
    class Meta(MainMeta):
        tablename = "contests"

    id: int = ormar.BigInteger(primary_key=True)
    description_link: str = ormar.String(max_length=300)
    description: str = ormar.Text()
    chat: Chat = ormar.ForeignKey(Chat)
    date_start: datetime = ormar.DateTime(default=datetime.utcnow())
    date_end: datetime = ormar.DateTime()

Query:
date_now = datetime.utcnow()
models.Contest.objects.filter(models.Contest.chat.id == channel.id, models.Contest.date_end >= date_now).all()

Error:
NotImplementedError: Don't know how to literal-quote value datetime.datetime(2021, 5, 28, 13, 10, 16, 807404)

**Comment by collerek** [comment, marked as answer]
Best to convert to str, something like: 
`Contest.objects.filter(Contest.chat.id == channel.id, Contest.date_end >= date_now.strftime("%Y-%m-%d %H:%M:%S")).all()`

Note also that if you declare your field like this it's gonna set a constant value for all instances:
```
date_start: datetime = ormar.DateTime(default=datetime.utcnow()) # <= constant - executed once on start-up
date_start: datetime = ormar.DateTime(default=datetime.utcnow) # <= pointer to callable (no parenthesis) - called on each initialization
```

**Reply by NikitaAleshnikov** [reply]
Thank you for answer and ormar)
